zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -351,6 +354,131 @@ def pre_detach_from_host(cls):
+            logger.debug("fuser command failed (rc=%d) for %s, skip in-use check", r, dev_path)
+            return
+
+        pids = re.findall(r'\d+', o)
+        active_details = []
+        for pid in pids:
+            try:
+                comm = linux.read_file("/proc/%s/comm" % pid)
+                comm = comm.strip() if comm else "unknown"
+            except Exception:
+                # Process vanished between fuser and read — skip it
+                continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)